pyexasol/connection.py

```diff
@@ -1206,16 +1209,31 @@ def _get_ws_options(self):
         }
 
         if self.options["encryption"]:
-            # Use "websocket_sslopt" argument to set custom SSL options, which are passed directly to WebSocket client constructor
-            if self.options["websocket_sslopt"]:
-                options["sslopt"] = self.options["websocket_sslopt"]
-            # OpenID tokens are normally used for Exasol SAAS
-            # Strict certificate verification is enabled by default
-            elif self.options["access_token"] or self.options["refresh_token"]:
+            # OpenID tokens are normally used for Exasol SAAS, so strict certificate
```

Comment:
See encryption.rst

Reply:
The text has the same issues as the text in encryption.txt, so we probably should fix it in the same way.

Reply:
I tried to update it based on the documentation linked in:
https://github.com/exasol/pyexasol/issues/174

The 2 follow-up questions I have are:
1. Is it known what certification verification values (NONE, REQUIRED) are allowed for the username + PAT combination?
    -> I couldn't find it in the documentation that I referenced, and I'm not sure how to discover/verify it.
2. With these updates, does this PR also close https://github.com/exasol/pyexasol/issues/174 ?

Reply:
Ok, I tried to fix it based on #174 documentation, but it might be better if we briefly discuss how to interpret/verify official documentation, especially if it looks incomplete/incorrect from your pov.

Reply:
The PAT is used as password